src/platform/silabs/ConnectivityManagerImpl_WIFI.cpp

```diff
@@ -220,6 +220,28 @@ void ConnectivityManagerImpl::_OnWiFiStationProvisionChange()
     DeviceLayer::SystemLayer().ScheduleWork(DriveStationState, NULL);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+CHIP_ERROR ConnectivityManagerImpl::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & SEDIntervalsConfig)
+{
+    // For now Wi-Fi uses DTIM power save mode
+    SEDIntervalsConfig.ActiveIntervalMS = chip::System::Clock::Milliseconds32(50);
+    SEDIntervalsConfig.IdleIntervalMS   = chip::System::Clock::Milliseconds32(300);
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::_SetSEDIntervalsConfig(const ConnectivityManager::SEDIntervalsConfig & intervalsConfig)
+{
+    // not required
+    return CHIP_NO_ERROR;
```

Comment:
why is this not used like with thread?

Reply:
thread enters slow polling mode to set the SED interval config. But for WiFi this depends upon the AP i.e., DTIM interval and wifi device will automatically take care of the packet

Reply:
This implies that the intervals are fixed and that the device cannot be more or less responsive depnding on its state.

The purpose off slow / fast polling is to be able to have long sleeps while being resposive when there is an active exchange.

Reply:
resolving this since it is based on AP end